src/main/java/com/atlassian/bitbucket/jenkins/internal/model/deployment/BitbucketDeploymentEnvironment.java

```diff
@@ -0,0 +1,144 @@
+package com.atlassian.bitbucket.jenkins.internal.model.deployment;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+import static org.apache.commons.lang3.StringUtils.stripToNull;
+
+/**
+ * The details of an environment that was deployed to.
+ *
+ * @since deployments
+ */
```

Comment:
`@JsonIgnoreProperties(ignoreUnknown = true)` ?

Reply:
You have it on the other ones. I think it is better to add it (given it is so easy to do), and that way all the new classes have them, and if we change the client in the future it is there and we don't accidentally ship a bug.